.moai/specs/SPEC-LOOP-TERM-001/plan.md

```
@@ -0,0 +1,205 @@
+  ↓
+user: selects option
+  ↓
+orchestrator: routes to next action deterministically
+```
+
+## 5. Risks and Mitigations
+
+| Risk | P | I | Mitigation |
+|------|---|---|-----------|
+| 4 workflow의 inconsistent 적용 | High | Medium | canonical reference를 단일 진실 출처로, individual override 최소화 |
+| design `§11`과 의미 충돌 | Low | High | 키 prefix 분리 (termination.* vs gan_loop.*) + design.md에 차이점 절 명시 |
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)